templates/content/GeoBlazor.Template.WebApp/GeoBlazor.Template.WebApp/Components/App.razor

```diff
@@ -0,0 +1,21 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="/" />
+    <link rel="stylesheet" href="bootstrap/bootstrap.min.css" />
+    <link rel="stylesheet" href="app.css" />
+
+    <link rel="stylesheet" href="GeoBlazor.Template.WebApp.styles.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <HeadOutlet />
+</head>
+
+<body>
+    <Routes />
```

Comment:
Adding `@rendermode=...` here I believe will make the app work.

Reply:
Or, probably better for a template, create a wrapper component where you can set the render mode for just the mapview, and add some comments there. See the .NET 8 branch for more info.